src/commands/upload_file.ts

```diff
@@ -121,6 +124,42 @@ function getSingleFile(parameters: ParametersHelper, parentFolderId: FolderID):
 	return [singleParameter];
 }
 
+async function getRemoteFile(
+	parameters: ParametersHelper,
+	parentFolderId: FolderID
+): Promise<UploadPathParameter[] | undefined> {
+	const remoteFilePath = parameters.getParameterValue(RemotePathParameter);
+
+	if (!remoteFilePath) {
+		return undefined;
+	}
+
+	const tempFolder = getTempFolder();
+	const destinationFileName = parameters.getRequiredParameterValue(DestinationFileNameParameter);
+
+	const { pathToFile, contentType } = await downloadFile(
+		remoteFilePath,
+		tempFolder,
+		destinationFileName,
+		(downloadProgress: number) => {
+			if (showProgressLog) {
+				process.stderr.write(`Downloading file... ${downloadProgress.toFixed(1)}% \r`);
```

Comment:
I'm getting an annoying console artifact due to the fact that we're using `write`:
```
yarn ardrive upload-file --remote-path "http://foo.com" --dest-file-name "wtvr.html"  -F 2d9386b7-25e5-40ea-a185-6a5508f8ed6a -w arweave-keyfile-9ODOd-_ZT9oWoRMVmmD4G5f9Z6MjvYxO3Nen-T5OXvU_cliSamplesmix.json --dry-run
{ownloading file... 0.0%
    "created": [

```
namely:
`{ownloading file... 0.0%`
You have 2 options:
1) Clear the line after the download is done:
```
    const { pathToFile, contentType } = await downloadFile(
		remoteFilePath,
		tempFolder,
		destinationFileName,
		(downloadProgress: number) => {
			if (showProgressLog) {
				process.stderr.write(`Downloading file... ${downloadProgress.toFixed(1)}% \r`);
			}
		}
	);
    process.stderr.clearLine(0);
```
2) Add a newline after the download, preserving the last downloading message:
```
    const { pathToFile, contentType } = await downloadFile(
		remoteFilePath,
		tempFolder,
		destinationFileName,
		(downloadProgress: number) => {
			if (showProgressLog) {
				process.stderr.write(`Downloading file... ${downloadProgress.toFixed(1)}% \r`);
			}
		}
	);
    process.stderr.write('\n');
```

Reply:
We should clear it so we have a clean output when tx are ready.

Reply:
It is actually writing to STDERR. In my opinion the latter (writing the line break) is the best approach, but instead of that we should print the whole line again (because at the end of the printed string there is a carriage return).

_Not a blocker, I'm fine with the current approach as well_

So @arielmelendez, to avoid that behavior you should redirect the STDERR output to null:
```sh
$ yarn ardrive upload-file --remote-path "http://foo.com" --dest-file-name "wtvr.html"  -F 2d9386b7-25e5-40ea-a185-6a5508f8ed6a -w arweave-keyfile-9ODOd-_ZT9oWoRMVmmD4G5f9Z6MjvYxO3Nen-T5OXvU_cliSamplesmix.json --dry-run 2> /dev/null
```